### ISSUE TITLE
Long-press shortcut support

### ISSUE THREAD

**Issue by lancelot-moon** [AdguardTeam/AdguardForiOS#1600]
AG for Android or Surge for iOS has such feature.

**Comment by rodaimi** [comment]
Are you referring to long pressing on the app icon? If so, you can currently long press on the icon to enable/disable protection. I would like to see two switces, mirroring the functionality of the existing widget. One to control DNS filtering, the other just for safari content blockers. Additionally, I would like to see a shortcut to the dns filtering log and the DNS protection settings. I find myself frequently needing to switch between AG dns and Google in order to unblock certain websites on my home network which is protected by AG DNS at the network level.

**Comment by lancelot-moon** [comment]
Yes, but AG for iOS still doesn’t give options via long-press shortcut.
You can suggest options which you wish to AG team...
